biggus/__init__.py

```diff
@@ -561,6 +563,175 @@ def _normalise_keys(self, keys):
         return keys
 
 
+class ArrayContainer(Array):
```

Comment:
Right ... but there's only one subclass. That suggests either don't bother, or refactor other class(es) to make use of it?